### PR TITLE
TINY-3731: Added docs about new accessibility checker settings

### PR DESCRIPTION
**Pull request by lnewson**

### CODE REVIEW

File: plugins/a11ychecker.md [hunk 1]
```diff
@@ -131,6 +131,52 @@ tinymce.init({
 });
 ```
 
+### `a11ychecker_html_version`
+
+This configuration option sets the HTML version to use when checking issues.
+
+For example, the "Complex tables should have summaries" rule in HTML4 the attribute is required because it's a valid attribute, however in HTML5 the attribute is deprecated, so the rule doesn't trigger.
```
Comment (wonderbugtiny) [inline plugins/a11ychecker.md]: Maybe something like this?
>  For example, setting the version to HTML4 will trigger the rule "Complex tables should have summaries" as summary is a valid attribute and is required for tables, however in HTML5 the attribute is deprecated so, the rule wouldn't be triggered.

Comment (lnewson) [reply plugins/a11ychecker.md]: Yeah, that's a lot better thanks!

File: plugins/a11ychecker.md [hunk 2]
```diff
@@ -131,6 +131,52 @@ tinymce.init({
...
+    selector: "textarea",
+    plugins: "a11ychecker",
+    toolbar: "a11ycheck",
+    a11ychecker_html_version: 'html5'
+});
+```
+
+### `a11ychecker_level`
+
+This configuration option sets the WCAG level to use when checking issues.
```
Comment (tylerkelly13) [inline plugins/a11ychecker.md]: `[WCAG level](https://www.w3.org/TR/WCAG20/#conformance)`

Comment (tylerkelly13) [reply plugins/a11ychecker.md]: not critical, but helpful

**Pull request merged**